aliBuild

```diff
@@ -10,7 +10,7 @@ from urllib2 import urlopen, URLError
 import ssl
 import json
 
-debug, error, riemannStream = (None, None, None)
+debug, error, warning, riemannStream = (None, None, None, None)
```

Comment:
Use a printout rather than a warning, since we want the warnings to be printed out also when we are not in debug mode.